### PR TITLE
Releases/v1.1.0

### PR DESCRIPTION
**Pull request by phil-cd**
This PR updates las2peer to work with Java 14 (corresponding issue: https://github.com/rwth-acis/las2peer/issues/64, corresponding PR into develop: https://github.com/rwth-acis/las2peer/pull/78).
Please note, that the update to Java 14 introduces new warning messages (see https://github.com/rwth-acis/las2peer/issues/82).
Besides that, the current las2peer version is now displayed in the statusbar of the node frontend (corresponding PR into develop: https://github.com/rwth-acis/las2peer/pull/81).

### CODE REVIEW

File: webconnector/frontend/false/_cacache/content-v2/sha512/0b/61/241d7c17bcbb1baee7094d14b7c451efecc7ffcbd92598a0f13d313cc9ebc2a07e61f007baf58fbf94ff9a8695bdd5cae7ce03bbf1e94e93613a00f25f21 [hunk 1]
```diff
@@ -0,0 +1 @@
+.
```
Comment (pedela) [inline webconnector/frontend/false/_cacache/content-v2/sha512/0b/61/241d7c17bcbb1baee7094d14b7c451efecc7ffcbd92598a0f13d313cc9ebc2a07e61f007baf58fbf94ff9a8695bdd5cae7ce03bbf1e94e93613a00f25f21:1]: Where do these come from?

File: webconnector/frontend/false/_cacache/index-v5/2e/66/d17574b72851a12c5b1cbd4c7b2c5d4205762cc3d80db67e6725bb515b20 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+
+d7ec3af497f1eb23fd0a5296434bcd485d04ac7b	{"key":"pacote:range-manifest:https://registry.npmjs.org/regenerator-runtime/-/regenerator-runtime-0.10.5.tgz:sha1-M2w+/BIgrc7dosn6tntaeVWjNlg=","integrity":"sha512-C2EkHXwXvLsbrucJTRS3xFHv7Mf/y9klmKDxPTE8yevCoH5h8Ae69Y+/lP+ahpW91crnzgO78elOk2E6APJfIQ==","time":1607423482007,"size":1,"metadata":{"id":"regenerator-runtime@0.10.5","manifest":{"name":"regenerator-runtime","version":"0.10.5","dependencies":{},"optionalDependencies":{},"devDependencies":{},"bundleDependencies":false,"peerDependencies":{},"deprecated":false,"_resolved":"https://registry.npmjs.org/regenerator-runtime/-/regenerator-runtime-0.10.5.tgz","_integrity":"sha1-M2w+/BIgrc7dosn6tntaeVWjNlg=","_shasum":"336c3efc1220adcedda2c9fab67b5a7955a33658","_shrinkwrap":null,"bin":null,"_id":"regenerator-runtime@0.10.5"},"type":"finalized-manifest"}}
```
Comment (pedela) [inline webconnector/frontend/false/_cacache/index-v5/2e/66/d17574b72851a12c5b1cbd4c7b2c5d4205762cc3d80db67e6725bb515b20:2]: this seems to be a new cache that was added? Why, and what does it do?
And in any case, it should not be in the repository, right? Could somebody clarify this?

**Review by pedela — CHANGES_REQUESTED** [review]
please clear the caches and log files before we can merge this

File: webconnector/frontend/false/_cacache/content-v2/sha512/0b/61/241d7c17bcbb1baee7094d14b7c451efecc7ffcbd92598a0f13d313cc9ebc2a07e61f007baf58fbf94ff9a8695bdd5cae7ce03bbf1e94e93613a00f25f21 (hunk 1 above)
Comment (Aran30) [reply webconnector/frontend/false/_cacache/content-v2/sha512/0b/61/241d7c17bcbb1baee7094d14b7c451efecc7ffcbd92598a0f13d313cc9ebc2a07e61f007baf58fbf94ff9a8695bdd5cae7ce03bbf1e94e93613a00f25f21:1]: A little mishap during one of my commits. Will delete the files

Comment (Aran30) [reply webconnector/frontend/false/_cacache/content-v2/sha512/0b/61/241d7c17bcbb1baee7094d14b7c451efecc7ffcbd92598a0f13d313cc9ebc2a07e61f007baf58fbf94ff9a8695bdd5cae7ce03bbf1e94e93613a00f25f21]: Was deleted in [0396a88](https://github.com/rwth-acis/las2peer/pull/83/commits/0396a88dbe52b8abf1e63c97ee5eca71e22b78ee)

**Review by pedela — APPROVED** [review]

**Pull request merged**